Google.PowerShell.IntegrationTests/Container/Container.GkeCluster.Tests.ps1

```diff
@@ -406,6 +406,87 @@ Describe "Add-GkeCluster" {
     }
 }
 
+Describe "Set-GkeCluster" {
+    $r = Get-Random
+    $script:clusterOneName = "gcp-set-gkecluster-1-$r"
+    $script:clusterOneZone = $zone
+
+    $script:clusterTwoName = "gcp-set-gkecluster-2-$r"
+    $script:clusterTwoZone = "asia-east1-a"
+
+    $script:clusterThreeName = "gcp-set-gkecluster-3-$r"
+    $script:clusterThreeZone = "europe-west1-b"
+
+    $script:clusterFourName = "gcp-set-gkecluster-4-$r"
+    $script:clusterFourZone = "us-west1-a"
+
+    $script:clusterNames = @($clusterOneName, $clusterTwoName, $clusterThreeName, $clusterFourName)
+    $script:clusterZones = @($clusterOneZone, $clusterTwoZone, $clusterThreeZone, $clusterFourZone)
```

Comment:
Some of the loops might be easier to write as a pipeline if these were a single array of name/zone pairs.